### ISSUE TITLE
So I downloaded the repo and did a node install

### ISSUE THREAD

**Issue by didlie** [m-onz/obmesh#2]
There were no errors in the build.

However 'build.js' was not constructed. Is this node or browser? I'm avoiding reading through code at the moment, but I want to do a quick test with a few nodes. 

Also, I added you to a team to build the official NoClouds website, if you feel like it, and if you have a good webapp platform in the barrel. Just pull the trigger and make a website and I will route noclouds.xyz to the github website.

And... what is your preffered method of communicating as a team? Do you want me to set up slack or create a whatapp group or something?

**Comment by m-onz** [comment]
Sorry to hear about you having issues getting this repo running. I'm super busy today but I will investigate this evening and update the README file with better instructions.

I normally make websites with http://tachyons.io/ and or https://choo.io/  (the same libz this repo uses).

Slack is fine for me.

**Comment by didlie** [comment]
I agree with your website platforms. They both look quick, easy. Feel free to push a simple website, if you want. slack channel : https://app.slack.com/client/TLQLYNFGT/CLQLHHSKU , welcome join link: https://join.slack.com/t/noclouds/shared_invite/enQtNzE2MTAzNTY1OTIwLWRlNDUxZWE1Y2FiZjk3YzA3ZTY1OTYxOWY0NGVkMmU5NzgyYTRmZWFmMDM4YzA2YzAzMzI2OGEwOWVmNTZlNzY , expires in 30 days
